scripts/version_mapper.py

```diff
@@ -80,14 +88,38 @@ def update_switch_version_file(filename, new_version, cname, render_last):
 
     # Override the whole content of the version switches JSON file with the new
     # generated version data
-    with open(f"release/{filename}", "w") as switcher_file:
+    with open(f"release/{json_filename}", "w") as switcher_file:
         # Update JSON file with the new content
         json.dump(new_content, switcher_file, indent=4)
 
+    # Use the latest stable verion for formatting the announcement
+    with open(annoucement_filename, "r") as announcement_file:
+        content = announcement_file.read()
+        announcement_content = content.format(version=latest_stable_version)
+
+    # Include the announcement in all available release folders. Note that
+    # these are still accessible even if they are not included in the dropdown.
+    release_folders = [path for path in Path("./").iterdir() if path.is_dir()]
+    for release_folder in release_folders:
+        # Skip the latest stable version
+        if str(release_folder) == latest_stable_version:
+            continue
```

Comment:
I was thinking, should we add announcement file for all the version, and on the latest release, the announcement will be "You are on the latest release" or any customized announcement ?

Reply:
I wouldn't do that. You only announce stuff if you have something to announce. If you are on the latest docs... I think there is no need to do that. I haven't seen that behaviour anywhere.